pkg/clustermanager/shim.go

```diff
@@ -16,32 +13,22 @@ import (
 // ClusterManager behavior to create a cluster for new workflows.
 type CreateClusterShim struct {
 	spec     *cluster.Spec
-	manager  *ClusterManager
 	provider providers.Provider
 }
 
 // NewCreateClusterShim returns a new CreateClusterShim instance.
 func NewCreateClusterShim(
 	spec *cluster.Spec,
-	manager *ClusterManager,
 	provider providers.Provider,
 ) *CreateClusterShim {
 	return &CreateClusterShim{
-		spec:    spec,
-		manager: manager,
+		spec: spec,
 	}
 }
 
 // CreateAsync satisfies the workload.Cluster interface.
 func (s CreateClusterShim) CreateAsync(ctx context.Context, management *types.Cluster) error {
-	if err := s.manager.applyProviderManifests(ctx, s.spec, management, s.provider); err != nil {
-		return fmt.Errorf("installing cluster creation manifests: %v", err)
-	}
-
-	if err := s.manager.InstallMachineHealthChecks(ctx, s.spec, management); err != nil {
-		return fmt.Errorf("installing machine health checks: %v", err)
-	}
-
+	// TODO: implement reusing the apply logic from clustermanager.Applier
```

Comment:
Do we still need these? This seems to be used in the "new workflows" that seemed to have never made it out of the feature flag